obiba_opal/system.py

```diff
@@ -364,31 +414,15 @@ def do_command(cls, args):
         """
         # Build and send request
         client = core.OpalClient.build(core.OpalClient.LoginInfo.parse(args))
-        try:
-            request = client.new_request()
-            request.fail_on_error()
+        service = RESTService(client, args.accept, args)
+        method = args.method if args.method else 'GET'
+        serviceMethod = getattr(service, 'send_%s_request' % method.lower())
```

Comment:
What happens if method is 'POTATOE'?

Reply:
See https://github.com/obiba/opal-python-client/blob/improvement/RESTService/obiba_opal/core.py#L281

Reply:
that's not what will happen: you are getting an object attribute that does not exist.

Reply:
Now it weill catch error:
```
AttributeError: 'RESTService' object has no attribute 'send_potato_request'
```